validator/main.py

```diff
@@ -106,46 +111,54 @@ def get_toxicity(self, value: str) -> List[str]:
     def validate_each_sentence(
         self, value: str, metadata: Dict[str, Any]
     ) -> ValidationResult:
-        """Validate that each sentence in the generated text is toxic."""
-
-        # Split the value into sentences using nltk sentence tokenizer.
         sentences = nltk.sent_tokenize(value)
 
-        unsupported_sentences, supported_sentences = [], []
+        toxic_sentences = []
+        non_toxic_sentences = []
+
         for sentence in sentences:
             if sentence:
                 pred_labels = self.get_toxicity(sentence)
                 if pred_labels:
-                    unsupported_sentences.append(sentence)
+                    toxic_sentences.append(sentence)
                 else:
-                    supported_sentences.append(sentence)
+                    non_toxic_sentences.append(sentence)
 
-        if unsupported_sentences:
-            unsupported_sentences = "- " + "\n- ".join(unsupported_sentences)
+        if toxic_sentences:
+            fixed_text = " ".join(non_toxic_sentences)
+            error_spans = self.get_error_spans(value, fixed_text)
+            toxic_sentences_text = "- " + "\n- ".join(toxic_sentences)
```

Comment:
Nitpick: Why the leading "- "?  The error_message at 136 is prepending a newline, so we could remove that and this "- " to leave just `"\n- ".join(...)`.

Reply:
I think an example of what this looks like when rich printed would be helpful.